playbook/app/pb_kits/playbook/pb_dialog/_dialog.jsx

```diff
@@ -55,7 +55,7 @@ const Dialog = (props: DialogProps) => {
   const dataProps = buildDataProps(data)
 
   const dialogClassNames = {
-    base: classnames('pb_dialog', buildCss('pb_dialog', size)),
+    base: classnames('pb_dialog', buildCss('pb_dialog', size), className ? buildCss('pb_dialog', className) : null),
```

Comment:
Thanks for the PR @stevelandwehr. At first glance I think there is probably a better way to write this with `classnames`. Take a look at: https://github.com/JedWatson/classnames#usage

`classnames('pb_dialog', buildCss('pb_dialog', size), { buildCss('pb_dialog', className): className })` potentially, but even that is a bit rough. @jasperfurniss Thoughts

Reply:
@stevelandwehr I just messaged you in connect over this.